docs/tutorial/01_bootstrap_juju.md

```diff
@@ -27,6 +29,13 @@ sudo snap install juju --channel=3.1/stable
 
 Bootstrap a Juju controller
 
+```{note}
+If you have never had Juju installed on your machine before, prior to bootstrapping 
+a Juju controller, from your terminal, create directories required by Juju:
+mkdir -p /home/ubuntu/.local/share
+This is a workaround to [a Juju bug](https://bugs.launchpad.net/juju/+bug/1988355).
```

Comment:
```suggestion
There is a [bug](https://bugs.launchpad.net/juju/+bug/1988355) in Juju that occurs when bootstrapping a controller on a new machine. If you encounter it, create the following directory:`mkdir -p /home/ubuntu/.local/share`.
```

And this comment should occur "after" the bootstrap in the sense that the user should do this only if the error has occured.

Reply:
:+1: